modules/rostests/apitests/wscript/wsf.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * PROJECT:     ReactOS API tests
+ * LICENSE:     LGPL-2.1+ (https://spdx.org/licenses/LGPL-2.1+)
+ * PURPOSE:     Test for wscript.exe
+ * COPYRIGHT:   ReactOS Team
+ */
+
+#include <apitest.h>
+#include <windows.h>
+#include <shlwapi.h>
+
+#define MYGUID "{898AC78E-BFC7-41FF-937D-EDD01E666707}"
+
+static DWORD getregdw(HKEY hKey, LPCSTR sub, LPCSTR name, DWORD *out, DWORD defval)
+{
+    DWORD size = sizeof(*out);
+    *out = 0;
+    LRESULT e = SHGetValueA(hKey, sub, name, NULL, out, &size);
+    if (e)
+        *out = defval;
+    return e;
+}
+
+static BOOL makestringfile(LPWSTR path, LPCSTR ext, LPCSTR string, const BYTE *map)
```

Comment:
Please make the function take the size of the path buffer in input, and have the ext as unicode.